modules/workflows/OrcaCopyToGlacierWorkflow/orca_copy_to_glacier_workflow.asl.json

```diff
@@ -9,7 +9,7 @@
           "task_config": {
             "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb}",
             "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
-            "collectionBucket": "{$.meta.collection.meta.orca_bucket}"
+            "orcaDefaultBucketOverride": "{$.meta.collection.meta.orca_default_bucket}"

```

Comment:
Need to change `"orcaDefaultBucketOverride": "{$.meta.collection.meta.orca_default_bucket}"` to 
`"orcaDefaultBucketOverride": "{$.meta.collection.meta.orcaDefaultBucketOverride}"` to be consistent with our other items in collection configuration.

Reply:
Fixed